### DISCUSSION TITLE
Requirement section or dependency

### DISCUSSION THREAD

**Discussion by RLee150** [CrowdStrike/falconpy#1217] [Ideas]
Hello,
New user to the Falconpy project. 
Does it make sense if users who are running falconpy in a virtualenv to self install missing modules?  Example - Running the online_since.py and failed due missing pandas.  I was able to pip install and fixed that issue.  But if users who may want a more seamless experience, a requirement page may help?

**Comment by jshcodes** [comment]
Hi @RLee150 -

Yes it does!  While some of our samples list requirements in the module docstring or sample documentation, this is not consistent. To address this, we have started adding `requirements.txt` files for each of the samples in the library so this is easier for developers. (Naming convention: _`samplename_requirements.txt`_, these files will reside in the same folder as the sample they describe.)

We are working on adding these now, so you will see them start to show up in folders as time progresses.
